File: Fundamentals C#/Basic Syntax, Conditional Statements and Loops - Exercise/04. Print and sum.cs
namespace _04._Print_and_sum
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int startNum = int.Parse(Console.ReadLine());
            int endNum = int.Parse(Console.ReadLine());
            double totalSum = 0;
            
            for (int i = startNum; i <= endNum; i++)
            {
                Console.Write($"{i} ");
                totalSum += i;
            }
            Console.WriteLine();
            Console.WriteLine($"Sum: {totalSum}");
        }
    }
}

File: JS-FrontEnd-Jan-2025/JS Front-End/JS Syntax Fundamentals - Exercises/04. Print And Sum.js
function sum(startNum, endNum) {
    let sum = 0;
    let numbers = '';

    for(let i = startNum; i <= endNum; i++) {
        numbers += i;
        numbers += ' ';
        sum += i; 
    }

    console.log(numbers);
    console.log(`Sum: ${sum}`);
}
